.circleci/config.yml

```diff
@@ -127,6 +154,9 @@ workflows:
           requires:
             - lint
             - validate
+      - test-bazel:
```

Comment:
it's not wise to have separate build and test jobs for bazel. you guarantee that the first test can't start running until the slowest build step is finished, so the total time is longer. Better to let bazel be the only scheduler for its actions

Reply:
Moved into the same job.